Neurotrauma/Lua/Scripts/Server/blood.lua

```diff
@@ -1,75 +1,60 @@
 -- Neurotrauma blood types functions
 -- Hooks Lua event "characterCreated" to create a randomized blood type for spawned character and sets their immunity to 100
----@diagnostic disable: lowercase-global, undefined-global
-NT.BLOODTYPE = { -- blood types and chance in percent
-    {"ominus",7},
-    {"oplus",37},
-    {"aminus",6},
-    {"aplus",36},
-    {"bminus",2},
-    {"bplus",8},
-    {"abminus",1},
-    {"abplus",3}}
-NT.setBlood = {}
-NT.foundAny = false
-
--- Insert all blood types in one table for RandomizeBlood()
-for index, value in ipairs(NT.BLOODTYPE) do
-    -- print(index," : ",value[1],", ",value[2],"%")
-    table.insert(NT.setBlood,index, {value[2],value[1]})
-end
-
--- Applies math.random() blood type.
--- returns the applied bloodtype as an affliction identifier
+---@diagnostic disable: undefined-global
+
+-- blood types and chance in percent
+local bloodTypeChance = {
+    { NTTypes.BloodType.o_minus,  7 },
+    { NTTypes.BloodType.o_plus,   37 },
+    { NTTypes.BloodType.a_minus,  6 },
+    { NTTypes.BloodType.a_plus,   36 },
+    { NTTypes.BloodType.b_minus,  2 },
+    { NTTypes.BloodType.b_plus,   8 },
+    { NTTypes.BloodType.ab_minus, 1 },
+    { NTTypes.BloodType.ab_plus,  3 }
```

Comment:
IMO its kinda weird having the blood types defined over in types.lua but the % chances defined separately over here in a non-moddable `local`.

I'll also note I'm probably the only dev who cares about any of this being moddable, since [NT Cybernetics Enhanced](https://github.com/Nebual/Neurotrauma/blob/cyberorgans/NT%20Cybernetics/Lua/Scripts/items.lua#L670) adds a 'C' type of Cyber-nanites-infected blood (which is effectively abc+), and strictly speaking NTCe doesn't care about the % chances, just seems weird to make them unmodifiable when they were before.

Reply:
I will then transfer this separately to the types of chances. Now